content/en/load-testing/Manual/300-test-setup.md

```diff
@@ -17,7 +17,7 @@ In case you want to switch the load between test runs for a quick check, the eas
 
 The load configuration usually lives in your `test.properties`. This file also contains the information which test cases are to be included in the load test run. Several of these test profile configuration files may be prepared in advance for different load test runs, then the needed properties are referenced for the current test run.
 
-Read more about [Load Configuration](../470-load-configuration). Read more about the load test profile configuration file [here](../480-test-suite-configuration/#load-test-profile-configuration).
+Read more about [Load Configuration](../470-load-configuration) and load test profile configuration file [here](../480-test-suite-configuration/#load-test-profile-configuration).
```

Comment:
suggestion: Read more about [Load Configuration](../470-load-configuration) and **the** [load test profile configuration file](../480-test-suite-configuration/#load-test-profile-configuration).